policy/scan/local_scanner.go

```diff
@@ -110,11 +149,13 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context) (*policy.Rep
 				return nil, false, errors.Wrap(err, "failed to generate a random asset MRN")
 			}
 			cur.Mrn = x.String()
+
+			// TODO: this is a hack, we need to implement the asset sync
+			cur.Mrn = "//assets.api.mondoo.app/spaces/test-infallible-taussig-796596/assets/2GBkbG6y7qeVtRsuIVBIsMpHGSM"
```

Comment:
This mrn needs to be determined during not yet implemented asset sync, for testing set update it manually